enterprise-suite/scripts/lbc.py

```diff
@@ -464,6 +464,21 @@ def install(creds_file):
 
     helm_args = ''
     if len(args.helm) > 0:
+        # namespace can be passed to helm directly after -- and can take format -namespace <value> or --namespace=<value>
+        # remove namespace from helm args and assign it args.namespace
+        # handle --namespace <val>
+        if  "--namespace" in args.helm and args.helm.index("--namespace") < len(args.helm):
+            ns_index=args.helm.index("--namespace")
+            args.namespace=args.helm[ns_index+1]
+            del args.helm[ns_index:ns_index+2]
+
+        # handle --namespace=<val>
+        r=re.compile("--namespace=.*")
+        if filter(r.match, args.helm):
```

Comment:
Should this be in an `else`?  Is there a point looking for the `--namespace=...` version if we've already found a bare `--namespace` version?

Reply:
marco I changed the logic, this comment is not relevant with the latest changes, please check